### DISCUSSION TITLE
eOS 8 Select Partitions Screen Disappeared During Installation

### DISCUSSION THREAD

**Discussion by WhiteHoot** [elementary/triage#649] [Q&A] [answered]
The title, the select partitions menu will disappear while getting the current configuration after a minute, i tried both safe mode and non safe mode.

And if i click modify partitions, the loading indicator would stop and just hangs.

I also successfully verified the iso.

My specs :
i5 12400
RTX 3070
32GB RAM
2TB SSD
3 Monitors, one of them is ultrawide

[Video](https://imgur.com/cgwNRWN)

**Comment by vjr** [comment]
This seems to be happening to at least a few more people (including me) too.

Do you have many partitions (in addition to maybe Windows partitions) already on the disk? Are you able to temporarily remove any additional unwanted partitions and see if the installer works?

One thing you could try is when the installer crashes/disappears then switch to TTY console (press CTRL+ALT+F1 or F2 etc) then login user `elementary` (no password) then run `sudo service lightdm restart` to bring the installer back up and try again, repeat this a few times to see if it works?

**Reply by WhiteHoot** [reply]
Correct, i do have several partitions including Windows partition because i dualboot, and no i don't have any unwanted partition, i also don't want to delete any partition rn since i can't successfully assign the partitions for eOS 8 cause i'll have to reinstall eOS 7.1 again if i erase current partitions.

I tried restarting lightdm when it crashed, no luck, screen went blank/black after.

**Comment by Alcizmax** [comment]
On the disk partition selection screen, there were hangs when loading "get current configuration after one minute" and in many cases the screen closed by itself.

**Comment by hemanthpai** [comment]
I am experiencing the same issue. I have a Thinkpad X1 Carbon 12th Gen laptop (Intel).

**Comment by hemanthpai** [comment]
Bringing up the TTY and restarting lightdm didn’t seem to work. I repeated those steps multiple times. However, I did following and was able to use the custom install option successfully (remember to connect to WiFi or plug in a LAN cable first)

`sudo apt-get update`

`sudo apt upgrade`

`sudo service lightdm restart`

**Reply by vverenich** [reply]
It worked! Thank you!

**Comment by alanuv7v** [comment]
Still having the same issue.

**Comment by Kernow66** [comment]
Same here.

**Comment by vjr** [comment]
Have you folks (re)downloaded the latest ISO from the website?

**Reply by Kernow66** [reply]
I have. I've tried the above method but still no luck. Funnily enough version 7 installs with no problem.

**Comment by HodurBalder** [comment]
same error 
<img width="1895" height="448" alt="Screenshot 2025-07-18 011001" src="https://github.com/user-attachments/assets/99e80922-4458-4508-9e30-4b6c460c2925" />

space of eOS 102.31GB 
i dont require swap 
only partition ext4 /
but Partitions Screen Disappeared

elementary OS 8.0.1 Circe 
elementaryos-8.0-stable.20250314rc.iso

**Comment by carlosss91** [comment]
I had this problem too, still waiting for the fix in new releases. Is there any fix? I think this should be a PRIORITY. If we can´t install it on the disk, how are we going to use it? Please, fix this now, it's been months from the issue!

**Comment by danirabbit** [comment, marked as answer]
We've released a series of updates to try to address this issue and we now have a fairly comprehensive test setup for layouts that cause this issue. If you can still reproduce this issue, please open a new issue report against the installer and list your partition layout so we can add it as a test case

https://github.com/elementary/installer/issues
